projects/rawkode.academy/platform/events-service/data-model/integrations/zod.ts

```diff
@@ -0,0 +1,81 @@
+import { eventsTable } from "../schema.ts";
+import { createInsertSchema } from "drizzle-zod";
+import { z } from "zod";
+
+export const EventStatus = z.enum(["scheduled", "cancelled", "rescheduled"]);
```

Comment:
I think `Scheduled` and `Cancelled` is enough. Reschedule'd events are still scheduled, it's just that the Date/Time has changed. Let's leave any special logic here for later, unless you feel this is mandatory now?